terraform/deploy/peering_concourse.tf

```diff
@@ -4,4 +4,13 @@ resource "aws_route" "route" {
   destination_cidr_block    = local.cidr_block[local.environment].mon-master-vpc
   vpc_peering_connection_id = module.vpc.outputs.prometheus_pcx
 }
-  
+
+resource "aws_security_group_rule" "web_lb_in_metrics" {
```

Comment:
You'll need the egress counterpart to this as well (allowing traffic out from the metrics SG to the concourse SG). Perhaps call this "concourse_metrics_ingress" and the egress one "metrics_concourse_egress"? This way, hopefully copy-pasting this file is easier as we can find/replace that `concourse` prefix :)

Reply:
As concourse is not initiating requests this should not be necessary

Reply:
Apologies @iw-gtadam I was too hasty in approving this; I don't think this is going to work without my suggested change.

You need 2 security groups:

1 - "egress" in the initiating SG so that traffic is allowed to leave it
2 - "ingress" to permit traffic into the receiving SG

We're still talking about a single "direction" of traffic here.

Reply:
@mattburgess I don't know what to say, it's a fact that it does work: https://monitoring.mgt-dev.dataworks.dwp.gov.uk/targets I have checked the security group rules on the concourse side and no egress rules on 9090 that could somehow allow it without me explicitly setting. https://github.com/dwp/aws-concourse/blob/master/terraform/modules/concourse_web/security_groups.tf

I get what you're saying but we should maybe address why it works without it before opening up potentially needlessly.